client/my-sites/stats/wordads/highlights-section.jsx

```diff
@@ -0,0 +1,154 @@
+import { Popover } from '@automattic/components';
+import { Icon, info, payment, receipt, tip } from '@wordpress/icons';
+import { numberFormat, translate } from 'i18n-calypso';
+import { useRef, useState } from 'react';
+import { useSelector } from 'react-redux';
+import { getWordAdsEarnings } from 'calypso/state/wordads/earnings/selectors';
+import HighlightCardSimple from './highlight-card-simple';
+
+// TODO: HighlightCard does not accept string values.
+// Should refactor to accept strings and move the business logic into the callers.
+// Then refactor this Comp to use HighlightCard again.
+
+function getAmountAsFormattedString( amount ) {
+	// Takes a Number, formats it to 2 decimal places, and prepends a "$".
+	// This mimics the existing behaviour. I'm assuming we only view/report
+	// on earnings in USD.
+	const formattedAmount = '$' + numberFormat( amount, 2 );
+	// Differs from previous behaviour in that we don't want "$0.00" as a result.
+	// Per design spec we'll return "$0" in this scenario.
+	// https://github.com/Automattic/wp-calypso/issues/72045
+	return formattedAmount === '$0.00' ? '$0' : formattedAmount;
+}
+
+function getHighlights( earnings ) {
+	const total = earnings && earnings.total_earnings ? Number( earnings.total_earnings ) : 0;
```

Comment:
You could use optional chaining to shorten this as `earnings?.total_earnings ? .... ...`.

And several places in the following code could be shortened in the same way.

Reply:
Thanks for the reminder. I was meaning to update the logic here. I'll get it updated.